src/index.ts

```diff
@@ -258,17 +258,24 @@ export interface MockCheckingAccount {
   routingNumber: string
 }
 
```

Comment:
Can you add that link to the spec as a comment here? I think it will be helpful to refer people there for more info.